config/default.json

```diff
@@ -2,6 +2,9 @@
   "authdb": {
     "baseuri": "http://127.0.0.1:8000/"
   },
+  "aws": {
+    "region": "us-east-1"
```

Comment:
I moved this to a general `aws` setting on the basis that there's no requirement to use queues in a different AWS region to the one we're sending email from. The auth server's ability to specify different regions for everything seems like something we'd never use here and just makes the config format noisier. But if I'm wrong about that, let me know and I can add separate `ses.region` and `sqs.region` settings.